<ui version="4.0" >
 <class>QProjectMConfigDialog</class>
 <widget class="QDialog" name="QProjectMConfigDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>449</width>
    <height>261</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>projectM Settings</string>
  </property>
  <widget class="QLabel" name="label" >
   <property name="geometry" >
    <rect>
     <x>12</x>
     <y>48</y>
     <width>60</width>
     <height>28</height>
    </rect>
   </property>
   <property name="sizePolicy" >
    <sizepolicy vsizetype="Preferred" hsizetype="MinimumExpanding" >
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize" >
    <size>
     <width>50</width>
     <height>0</height>
    </size>
   </property>
   <property name="text" >
    <string>Font Path</string>
   </property>
  </widget>
  <widget class="QToolButton" name="toolButton" >
   <property name="geometry" >
    <rect>
     <x>404</x>
     <y>48</y>
     <width>30</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text" >
    <string>...</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="fontPathLineEdit" >
   <property name="geometry" >
    <rect>
     <x>78</x>
     <y>49</y>
     <width>320</width>
     <height>25</height>
    </rect>
   </property>
   <property name="sizePolicy" >
    <sizepolicy vsizetype="Fixed" hsizetype="Expanding" >
     <horstretch>2</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
  </widget>
  <widget class="QLabel" name="label_2" >
   <property name="geometry" >
    <rect>
     <x>12</x>
     <y>12</y>
     <width>96</width>
     <height>28</height>
    </rect>
   </property>
   <property name="sizePolicy" >
    <sizepolicy vsizetype="Preferred" hsizetype="MinimumExpanding" >
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize" >
    <size>
     <width>50</width>
     <height>0</height>
    </size>
   </property>
   <property name="text" >
    <string>Startup Playlist</string>
   </property>
  </widget>
  <widget class="QToolButton" name="toolButton_2" >
   <property name="geometry" >
    <rect>
     <x>404</x>
     <y>12</y>
     <width>30</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text" >
    <string>...</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="startupPlaylistLineEdit" >
   <property name="geometry" >
    <rect>
     <x>114</x>
     <y>13</y>
     <width>284</width>
     <height>25</height>
    </rect>
   </property>
   <property name="sizePolicy" >
    <sizepolicy vsizetype="Fixed" hsizetype="Expanding" >
     <horstretch>2</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="toolTip" >
    <string>&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;style type="text/css">
p, li { white-space: pre-wrap; }
&lt;/style>&lt;/head>&lt;body style=" font-family:'Sans Serif'; font-size:9pt; font-weight:400; font-style:normal;">
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">This sets the default preset playlist when qprojectM starts up. This can be either a directory path or preset playlist file. If unspecified then projectM uses a default preset directory&lt;/p>&lt;/body>&lt;/html></string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3" >
   <property name="geometry" >
    <rect>
     <x>12</x>
     <y>84</y>
     <width>121</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text" >
    <string>Preset Duration (s)</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="presetDurationSpinBox" >
   <property name="geometry" >
    <rect>
     <x>138</x>
     <y>84</y>
     <width>54</width>
     <height>25</height>
    </rect>
   </property>
   <property name="toolTip" >
    <string>&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;style type="text/css">
p, li { white-space: pre-wrap; }
&lt;/style>&lt;/head>&lt;body style=" font-family:'Sans Serif'; font-size:9pt; font-weight:400; font-style:normal;">
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">Sets how long a preset should be played in seconds before projectM automatically switches to another in the playlist&lt;/p>&lt;/body>&lt;/html></string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4" >
   <property name="geometry" >
    <rect>
     <x>199</x>
     <y>84</y>
     <width>176</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text" >
    <string>Smooth Preset Duration (s)</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="smoothPresetDurationSpinBox" >
   <property name="geometry" >
    <rect>
     <x>380</x>
     <y>84</y>
     <width>54</width>
     <height>25</height>
    </rect>
   </property>
   <property name="toolTip" >
    <string>&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;style type="text/css">
p, li { white-space: pre-wrap; }
&lt;/style>&lt;/head>&lt;body style=" font-family:'Sans Serif'; font-size:9pt; font-weight:400; font-style:normal;">
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">Sets how long a transition occurs in seconds when smooth preset switching&lt;/p>&lt;/body>&lt;/html></string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5" >
   <property name="geometry" >
    <rect>
     <x>12</x>
     <y>117</y>
     <width>115</width>
     <height>25</height>
    </rect>
   </property>
   <property name="sizePolicy" >
    <sizepolicy vsizetype="Preferred" hsizetype="MinimumExpanding" >
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text" >
    <string>Initial Window Size</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6" >
   <property name="geometry" >
    <rect>
     <x>195</x>
     <y>117</y>
     <width>91</width>
     <height>25</height>
    </rect>
   </property>
   <property name="sizePolicy" >
    <sizepolicy vsizetype="Preferred" hsizetype="MinimumExpanding" >
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text" >
    <string>Maximum FPS</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="maxFPSSpinBox" >
   <property name="geometry" >
    <rect>
     <x>292</x>
     <y>117</y>
     <width>142</width>
     <height>25</height>
    </rect>
   </property>
   <property name="sizePolicy" >
    <sizepolicy vsizetype="Fixed" hsizetype="MinimumExpanding" >
     <horstretch>2</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="toolTip" >
    <string>&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;style type="text/css">
p, li { white-space: pre-wrap; }
&lt;/style>&lt;/head>&lt;body style=" font-family:'Sans Serif'; font-size:9pt; font-weight:400; font-style:normal;">
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">This parameter, if nonzero, limits the number of frames per second projectM will attempt to render to screen. 35 is a reasonable limit.&lt;/p>&lt;/body>&lt;/html></string>
   </property>
  </widget>
  <widget class="QSpinBox" name="initialWindowSizeSpinBox" >
   <property name="geometry" >
    <rect>
     <x>134</x>
     <y>117</y>
     <width>54</width>
     <height>25</height>
    </rect>
   </property>
   <property name="toolTip" >
    <string>&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;style type="text/css">
p, li { white-space: pre-wrap; }
&lt;/style>&lt;/head>&lt;body style=" font-family:'Sans Serif'; font-size:9pt; font-weight:400; font-style:normal;">
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">Sets how long a preset should be played in seconds before projectM automatically switches to another in the playlist&lt;/p>&lt;/body>&lt;/html></string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10" >
   <property name="geometry" >
    <rect>
     <x>334</x>
     <y>150</y>
     <width>16</width>
     <height>23</height>
    </rect>
   </property>
   <property name="sizePolicy" >
    <sizepolicy vsizetype="Preferred" hsizetype="MinimumExpanding" >
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text" >
    <string>by</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7" >
   <property name="geometry" >
    <rect>
     <x>12</x>
     <y>150</y>
     <width>77</width>
     <height>23</height>
    </rect>
   </property>
   <property name="sizePolicy" >
    <sizepolicy vsizetype="Preferred" hsizetype="MinimumExpanding" >
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text" >
    <string>Texture Size</string>
   </property>
  </widget>
  <widget class="QComboBox" name="textureSizeComboBox" >
   <property name="geometry" >
    <rect>
     <x>95</x>
     <y>150</y>
     <width>77</width>
     <height>23</height>
    </rect>
   </property>
   <property name="sizePolicy" >
    <sizepolicy vsizetype="Fixed" hsizetype="MinimumExpanding" >
     <horstretch>1</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="sizeAdjustPolicy" >
    <enum>QComboBox::AdjustToMinimumContentsLength</enum>
   </property>
  </widget>
  <widget class="QComboBox" name="meshSizeYComboBox" >
   <property name="geometry" >
    <rect>
     <x>355</x>
     <y>150</y>
     <width>79</width>
     <height>23</height>
    </rect>
   </property>
   <property name="sizePolicy" >
    <sizepolicy vsizetype="Fixed" hsizetype="MinimumExpanding" >
     <horstretch>1</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
  </widget>
  <widget class="QLabel" name="label_8" >
   <property name="geometry" >
    <rect>
     <x>178</x>
     <y>150</y>
     <width>65</width>
     <height>23</height>
    </rect>
   </property>
   <property name="sizePolicy" >
    <sizepolicy vsizetype="Preferred" hsizetype="MinimumExpanding" >
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text" >
    <string>Mesh Size</string>
   </property>
  </widget>
  <widget class="QComboBox" name="meshSizeXComboBox" >
   <property name="geometry" >
    <rect>
     <x>249</x>
     <y>150</y>
     <width>79</width>
     <height>23</height>
    </rect>
   </property>
   <property name="sizePolicy" >
    <sizepolicy vsizetype="Fixed" hsizetype="MinimumExpanding" >
     <horstretch>1</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
  </widget>
  <widget class="QCheckBox" name="fullscreenOnStartupCheckBox" >
   <property name="geometry" >
    <rect>
     <x>12</x>
     <y>181</y>
     <width>208</width>
     <height>24</height>
    </rect>
   </property>
   <property name="sizePolicy" >
    <sizepolicy vsizetype="Fixed" hsizetype="MinimumExpanding" >
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text" >
    <string>Fullscreen on Startup</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="menuOnStartupCheckBox" >
   <property name="geometry" >
    <rect>
     <x>226</x>
     <y>181</y>
     <width>208</width>
     <height>24</height>
    </rect>
   </property>
   <property name="sizePolicy" >
    <sizepolicy vsizetype="Fixed" hsizetype="MinimumExpanding" >
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text" >
    <string>Show Menu on Startup</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9" >
   <property name="geometry" >
    <rect>
     <x>12</x>
     <y>213</y>
     <width>206</width>
     <height>38</height>
    </rect>
   </property>
   <property name="sizePolicy" >
    <sizepolicy vsizetype="Preferred" hsizetype="MinimumExpanding" >
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text" >
    <string>&lt;B>WARNING&lt;/B>: All setting changes require a restart to take effect!</string>
   </property>
   <property name="textFormat" >
    <enum>Qt::AutoText</enum>
   </property>
   <property name="wordWrap" >
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QDialogButtonBox" name="buttonBox" >
   <property name="geometry" >
    <rect>
     <x>229</x>
     <y>217</y>
     <width>205</width>
     <height>29</height>
    </rect>
   </property>
   <property name="orientation" >
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons" >
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::NoButton|QDialogButtonBox::Ok</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
